src/hotspot/share/gc/shared/gcLocker.cpp

```diff
@@ -97,6 +107,12 @@ bool GCLocker::check_active_before_gc() {
   if (is_active() && !_needs_gc) {
     verify_critical_count();
     _needs_gc = true;
+#if INCLUDE_JFR
+    if (EventGCLocker::is_enabled()) {
+      _needs_gc_start_timestamp = JfrTicks::now();
```

Comment:
Do you really need to use JfrTicks instead of Ticks here? If not, could you remove all references and includes of JfrTicks? We usually use pass in Ticks when we send JFR events.

Reply:
Fixed